src/Dialect/ONNX/ONNXOps/Additional/LayoutTransform.cpp

```diff
@@ -43,8 +43,9 @@ LogicalResult ONNXLayoutTransformOp::inferShapes(
 //===----------------------------------------------------------------------===//
 LogicalResult ONNXLayoutTransformOp::verify() {
   if (hasShapeAndRank(getData()) && hasShapeAndRank(getOutput())) {
```

Comment:
@hamptonm1 this line looks redundant. It checks that both data and output must be shaped and ranked.

Reply:
Okay I can remove this line...it was leftover from the previous code. Let me remove then.